presto-main/src/main/java/com/facebook/presto/operator/scalar/ColorFunctions.java

```diff
@@ -149,8 +152,9 @@ public static long color(@SqlType(StandardTypes.DOUBLE) double fraction, @SqlTyp
     }
 
     @ScalarFunction
-    @SqlType(StandardTypes.VARCHAR)
-    public static Slice render(@SqlType(StandardTypes.VARCHAR) Slice value, @SqlType(ColorType.NAME) long color)
+    @LiteralParameters("x")
+    @SqlType(VarcharType.VARCHAR_MAX_LENGTH)
```

Comment:
I think it is possible to determine more precise output type. Ansi encoded color is fixed length value. So the result type should be `varchar(x + ansi_encoded_color_length_in_characters)`